xarray/backends/rasterio_.py

```diff
@@ -272,7 +280,14 @@ def open_rasterio(filename, parse_coordinates=None, chunks=None, cache=None, loc
     if lock is None:
         lock = RASTERIO_LOCK
 
-    manager = CachingFileManager(rasterio.open, filename, lock=lock, mode="r")
+    manager = CachingFileManager(
+        rasterio.open,
+        filename,
+        lock=lock,
+        mode="r",
+        backend_kwargs=backend_kwargs,
+        **kwargs,
```

Comment:
you would have to append `backend_kwargs` to `kwargs` (using `.update`) and then do
```suggestion
        kwargs=kwargs,
```

Reply:
Ok I'll just do this. Thanks @keewis